src/components/StudyPane/Header/Title.tsx

```diff
@@ -5,56 +5,63 @@ import { updateStudyName } from '@/lib/actions';
+  study: StudyData,
+  inViewMode: boolean
 }) => {
-    const [title, setTitle] = useState(study.name);
-    const [isEditing, setIsEditing] = useState(false);
+  const [title, setTitle] = useState(study.name);
+  const [isEditing, setIsEditing] = useState(false);
 
-    const titleRef = useRef<HTMLDivElement>(null);
-    const MAX_TITLE_LENGTH = 40;
-  
-    const handleSaveClick = useCallback(() => {
-      setIsEditing(false);
-      updateStudyName(study.id, title);
-    }, [study.id, title]);
-        
-    useEffect(() => {
-        const handleClickOutside = (event: MouseEvent) => {
-          // when clicked outside of the text box save the change
-          if (titleRef.current && !titleRef.current.contains(event.target as Node)) {
-              handleSaveClick();
-          }
-        };
-    
-        if (isEditing) {
-          document.addEventListener('mousedown', handleClickOutside);
-        } else {
-          document.removeEventListener('mousedown', handleClickOutside);
-        }
-    
-        return () => {
-          document.removeEventListener('mousedown', handleClickOutside);
-        };
-      }, [isEditing, title, handleSaveClick]);
+  const titleRef = useRef<HTMLDivElement>(null);
+  const MAX_TITLE_LENGTH = 40;
 
-    const handleInputChange = (event: ChangeEvent<HTMLInputElement>) => {
-      setTitle(event.target.value);
+  const handleSaveClick = useCallback(() => {
+    setIsEditing(false);
+    updateStudyName(study.id, title);
+  }, [study.id, title]);
+      
+  useEffect(() => {
+    const handleClickOutside = (event: MouseEvent) => {
+      // when clicked outside of the text box save the change
+      if (titleRef.current && !titleRef.current.contains(event.target as Node)) {
+          handleSaveClick();
+      }
     };
 
-  
-    const handleEditClick = () => {
-      setIsEditing(true);
+    if (isEditing) {
+      document.addEventListener('mousedown', handleClickOutside);
+    } else {
+      document.removeEventListener('mousedown', handleClickOutside);
+    }
+
+    return () => {
+      document.removeEventListener('mousedown', handleClickOutside);
     };
+  }, [isEditing, title, handleSaveClick]);
 
-    function truncateString(str: string, maxLength: number): string {
-      if (str.length <= maxLength) {
-          return str;
-      } else {
-          return str.slice(0, maxLength) + '...';
-      }
+  const handleInputChange = (event: ChangeEvent<HTMLInputElement>) => {
+    setTitle(event.target.value);
+  };
+
+  const handleEditClick = () => {
+    setIsEditing(true);
+  };
+
+  function truncateString(str: string, maxLength: number): string {
+    if (str.length <= maxLength) {
+        return str;
+    } else {
+        return str.slice(0, maxLength) + '...';
+    }
   }
-  
+
+  if (inViewMode) { // render static title
+    return (
+      <div className="block">
+        <h1 className="text-title-sm" >{truncateString(study.name, MAX_TITLE_LENGTH)}</h1>
+      </div>
+    );
```

Comment:
Instead of adding a duplicate block of rendering code, can you just disable the onClick in Line 81 when it's in view mode?